system/lib/compiler-rt/lib/sanitizer_common/sanitizer_linux.cc

```diff
@@ -207,7 +208,8 @@ int internal_mprotect(void *addr, uptr length, int prot) {
 #endif
 
 uptr internal_close(fd_t fd) {
-  return internal_syscall(SYSCALL(close), fd);
+  // XXX EMSCRIPTEN: use wasi
```

Comment:
"use wasi" As in, this should be `#ifdef __EMSCRIPTEN__`?

Reply:
Thanks, yeah, I'll make this consistent with the rest.